app/renderer/utils/other.js

```diff
@@ -28,3 +28,36 @@ export function addVendorPrefixes(caps) {
     return cap;
   });
 }
+
+export function pixelsToPercentage(px, maxPixels) {
+  if (!isNaN(px)) {
+    return parseFloat(((px / maxPixels) * 100).toFixed(1), 10);
+  }
+}
+
+export function percentageToPixels(pct, maxPixels) {
+  if (!isNaN(pct)) {
+    return Math.round(maxPixels * (pct / 100));
+  }
+}
+
+export function parseCoordinates(element) {
+  let {bounds, x, y, width, height} = element.attributes || {};
```

Comment:
why cannot we use const?

Reply:
I didn't modify these methods, but some variables can indeed be converted to consts. Not these though - only `bounds` is not changed. We can however just create new const variables, allowing to change these to consts too.